### PR TITLE
feat: [PL-23879]: No Expiration in Token Form

### PR DESCRIPTION
**Pull request by reetika04**
##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by reetika04** [comment]
Screenshot 2022-04-01 at 5 33 21 PM
Screenshot 2022-04-01 at 5 33 33 PM

**Review by harshilbits — DISMISSED** [review]

File: src/modules/20-rbac/modals/TokenModal/views/EditToken.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import { defaultTo, omit } from 'lodash-es'
+import { useToaster } from '@common/components'
+import { useStrings } from 'framework/strings'
+import { useUpdateToken } from 'services/cd-ng'
+import { getRBACErrorMessage } from '@rbac/utils/utils'
+import TokenForm, { TokenFormData } from './TokenForm'
+import type { TokenModalProps } from './CreateToken'
+
+const EditTokenForm: React.FC<TokenModalProps> = props => {
+  const { data: tokenData, onSubmit, onClose } = props
```
Comment (swarajcheguri-harness) [inline src/modules/20-rbac/modals/TokenModal/views/EditToken.tsx:19]: cant we resue createToken for edit token too ?

**Review by swarajcheguri-harness — APPROVED** [review]

**Comment by reetika04** [comment]
trigger cypress

**Comment by reetika04** [comment]
trigger rebase

**Review by swarajcheguri-harness — DISMISSED** [review]

**Review by swarajcheguri-harness — APPROVED** [review]

**Pull request merged**